packages/react/src/River/River.module.css

```diff
@@ -41,6 +42,9 @@
 @media screen and (min-width: 34rem) {
   .River {
     grid-template-rows: unset;
+    gap: var(--brand-River-medium-gap);
+    padding-top: var(--brand-River-medium-padding);
+    padding-bottom: var(--brand-River-medium-padding);
```

Comment:
Our medium breakpoint is `48rem`. could we add these to a new media query or update the value to 48rem?

Reply:
done 👍